trio/_socket.py

```diff
@@ -338,19 +418,21 @@ def _sniff_sockopts_for_fileno(family, type, proto, fileno):
 )
 
 
-def _make_simple_sock_method_wrapper(methname, wait_fn, maybe_avail=False):
-    fn = getattr(_stdlib_socket.socket, methname)
-
+def _make_simple_sock_method_wrapper(
+    fn: Callable[Concatenate[_stdlib_socket.socket, P], T],
+    wait_fn: Callable,
```

Comment:
Any reason `wait_fn` isn't fully typed?

Reply:
nope, especially when it is in `_nonblocking_helper` - I just forgot. I should enable https://mypy.readthedocs.io/en/stable/config_file.html#confval-disallow_any_generics